eth/vm/computation.py

```diff
@@ -516,7 +525,7 @@ def __exit__(self, exc_type: None, exc_value: None, traceback: None) -> None:
                 self.msg.value,
                 self.msg.depth,
                 "y" if self.msg.is_static else "n",
-                self.msg.gas - self._gas_meter.gas_remaining,
+                self.get_gas_used(),
```

Comment:
This should be the only functionality change.

Reply:
So, the previous implementation did not account for `should_burn_gas`? Was that wrong? I can't tell but I guess you know what you're doing :)

Reply:
This is a logging message so it was likely logging the wrong value in cases where gas was not burned.